server/src/main/java/com/linecorp/centraldogma/server/internal/api/GitHttpService.java

```diff
@@ -113,10 +118,10 @@ public HttpResponse advertiseCapability(@Header("git-protocol") @Nullable String
             return HttpResponse.of(HttpStatus.NOT_FOUND, MediaType.PLAIN_TEXT_UTF_8,
                                    "Project not found: " + projectName);
         }
-    if (!projectApiManager.getProject(projectName).repos().exists(repoName)) {
+        if (!projectApiManager.getProject(projectName, user).repos().exists(repoName)) {
```

Comment:
Question) Is it possible to just use the user in the thread when possible?  The current context/user seems like a cross cutting concern, and I'm not sure if it is practical to change signatures whenever we need to access a user from a new location.

e.g.
```suggestion
        if (!projectApiManager.getProject(projectName).repos().exists(repoName)) {
```

Reply:
I wasn't sure all executors that Central Dogma uses are context-aware. Initially, I was thinking of using thread context, but I also thought there might be different opinions about this. When considering the two patterns, I concluded the explicit one had more advantages.